src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/SelectVariants.java

```diff
@@ -420,6 +423,9 @@
             doc="Suppress reference path in output for test result differencing")
     private boolean suppressReferencePath = false;
 
+    @ArgumentCollection
+    private GenomicsDBArgumentCollection genomicsdbArgs = new GenomicsDBArgumentCollection();
```

Comment:
Has any thought been given to how this new argument interacts with the existing genotype subsetting arguments SelectVariants already has ? If not, I'm thinking this change will likely cause confusion, and that we should take this out until those are resolved and some tests are added.

Reply:
There is no existing argument that clashes with the existing subsetting arguments. However, if you would rather take this out, please let me know.

Reply:
SV has several arguments that do various kinds of genotype subsetting/downsampling/filtering. The argument is marked as `@Advanced`, so maybe its ok. But if this argument does what I think it does, it may not be obvious to a user that this subsetting is done independently before SV ever sees the data.

Reply:
This needs more thought. Have removed all support for GenomicsDB args in SelectVariants for now and have filed #6456 to generally get the tools hook up to GenomicsDBExportConfiguration better.